samples: Bluetooth: Direction finding changes in configuration

ppryga-nordic opened the pull request:
- samples: Bluetooth: DF add configuration for nRF54LXXX SoCs
    
  Add configuration for direction finding samples that showcase
  DF functionalities supported by SoftDevice Contoller.
    
  Cleanup comment in existing configurations.

- samples: Bluetooth: Make the connectionless TX AoD disabled by default
    
  Change the configuration of DF connectionless TX sample to disable
  the TX AoD (antenna switching TX) by default. The configuration
  is available when the sample is build with the Zephyr's BLE
  controller.

- samples: Bluetooth: DF Tx antenna switching should be disabled in ctrl
    
  The CONFIG_BT_CTLR_DF_ANT_SWITCH_TX belongs to BLE controller
  configuration. It should be cleared in board files to make
  sure it is applied when the controller is complied.

richabp reviewed (commented):
Should there be a respective Changelog entry?

ppryga-nordic commented on doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:
```diff
@@ -376,6 +376,15 @@ Bluetooth samples
+
+* :ref:`direction_finding_connectionless_tx` sample:
+
+  * Added support for: ``nrf54l15dk/nrf54l15/cpuapp``, ``nrf54l15dk/nrf54l05/cpuapp``, ``nrf54lm20pdk/nrf54lm20a/cpuapp``, ``nrf54lv10dk/nrf54lv10a/cpuapp`` board targets.
```
Corrected

ppryga-nordic commented on doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:
```diff
@@ -376,6 +376,15 @@ Bluetooth samples
+
+* :ref:`direction_finding_connectionless_tx` sample:
+
+  * Added support for: ``nrf54l15dk/nrf54l15/cpuapp``, ``nrf54l15dk/nrf54l05/cpuapp``, ``nrf54lm20pdk/nrf54lm20a/cpuapp``, ``nrf54lv10dk/nrf54lv10a/cpuapp`` board targets.
```
Ah there is missing board overlay for that target. The target is listed in `sample.yaml`.

-> corrected

ppryga-nordic commented:
Corrected release notes.
Added missing board configs `nrf54l15dk_nrf54l10_cpuapp.conf`
Added missing entries in `sample.yaml` in `direction_finding_peripheral`
